libs/auth/src/card.ts

```diff
@@ -92,6 +92,13 @@ interface CheckPinResponseIncorrect {
   numIncorrectPinAttempts: number;
 }
 
+/**
+ * The base API any card should support
+ */
+export interface BaseCard {
+  disconnect(): Promise<void>;
+}
```

Comment:
nit: Could you move this below `CheckPinResponse` so that `CheckPinResponse` comes right after `CheckPinResponseCorrect` and `CheckPinResponseIncorrect`?